readme.md

```diff
@@ -524,6 +524,53 @@ test.only.serial(...);
 
 This means you can temporarily add `.skip` or `.only` at the end of a test or hook definition without having to make any other changes.
 
+### Test macros
+
+Additional arguments passed to the test declaration will be passed to the test implementation. This is useful for creating reusable test macros.
+
+```js
+function macro(t, input, expected) {
```

Comment:
Yeah, eating my words on that. Though not as badly as I've been forced to [eat these ones](https://github.com/sindresorhus/ava/issues/222#issuecomment-160546933)